gradle/dependencies.gradle

```diff
@@ -61,6 +61,7 @@ versions += [
   bcpkix: "1.66",
   checkstyle: "8.36.2",
   commonsCli: "1.4",
+  dropwizardMetrics: "3.2.5",
```

Comment:
Does this conflict with the yammer metrics library we use in Kafka?

Reply:
> Does this conflict with the yammer metrics library 

It doesn't, no.  The yammer metrics library is called `metrics-core-2.2.0.jar` and has all classes underneath the `com.yammer.metrics` package.  The dropwizard library is called `metrics-core-3.2.5.jar` and has all classes underneath the `com.codahale.metrics` package.  It isn't ideal that the jar names differ only by the version, but there is no conflict.

Reply:
Why are we using this old version? The latest version is 4.2.2.

Reply:
https://github.com/apache/zookeeper/blob/release-3.6.3/pom.xml#L367

```
<dropwizard.version>3.2.5</dropwizard.version>
```

Reply:
3.7 uses a newer version though, have we checked if we have to stick with this old version? It will be harder to update  in the 3.0 series after that's released.

Reply:
> have we checked if we have to stick with this old version

No.  How would we do that check?  I can think of 2 ways: 1) Feels too risky: just take the new version and call it good if the system tests pass; 2) Ask in the Zk mailing list maybe?  Any other options?

> It will be harder to update in the 3.0 series after that's released

It feels to me like the straightforward thing to do is use the version that ZK uses, and there's no need to update the dropwizard version unless we update the ZK version.  And if we are sticking with ZK 3.6.x through the EOL of ZooKeeper with Kafka (unknown, but potentially the case) then there's no reason to change it sans some kind of CVE.  Maybe I'm missing something, though?

Reply:
The straightforward way would be to check the version bump in the 3.7 branch, did that require code changes?

CVEs are the common reason to have to update these days.

Reply:
https://github.com/apache/zookeeper/commit/13fe0d0ffb9fd2c379b9b430aaaf9ee75acfceba

It's not the only version update that was made, but no code changes.

Reply:
https://github.com/apache/kafka/pull/10982 contains this change.  Will attache system test results there when tests complete.